doc/source/examples/integrated_gradients_transformers.ipynb

```diff
@@ -43,18 +43,22 @@
     "import numpy as np\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #80.        # see transformers issue #14404: https://github.com/huggingface/transformers/issues/14404
</pre><p>It seem the issue says this has been resolved now? Although probably would need a minimum version bump (not sure which one) in <code>alibi</code> for <code>transformers</code> to avoid people using <code>numpy</code> arrays with older versions of  <code>transformers</code> .   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/717/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>